modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganizationForm.testcase

```diff
@@ -68,7 +68,7 @@ definition {
 				zipCode = "12.345-67");
 		}
 
-		task ("When the employee user create a request") {
+		task ("When the Employee user create a request") {
```

Comment:
Just a typo here, could you change this task to: `"When the Employee user creates a request"`